apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -197,12 +197,17 @@ class ShareAllowedDialog extends React.Component {
     const tweetText = artistTwitterHandle
       ? `Check out the dance I made featuring @${artistTwitterHandle} on @codeorg!`
       : 'Check out what I made on @codeorg!';
+    const hashtags = this.props.selectedSong
+      ? ['codeplay', 'HourOfCode']
```

Comment:
Just to confirm, we want this to appear on all Dance Party shares, not just the Coldplay ones, right?

Reply:
Ideally we only want to have #codeplay for the Coldplay songs. @breville was that not possible?

Reply:
I've updated the PR so that `#codeplay` is only added for songs with artist `"Coldplay"`.